tests/unittests/report/list_conf_files_test.py

```diff
@@ -0,0 +1,73 @@
+"""Test for listing the report config files from a dir"""
```

Comment:
Come to think of it, this doesn't belong under `unittests/report`, as it does not test functionality from the report subsystem: It tests functionality from the `nav.config` module.

I.e. the test code should actually belong in `unittests/general/config_test.py`, where the other tests for this module is located.

Reply:
(why there is a `general` subdirectory there at all is an entirely different discussion, which we should have at some point :-D)